openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -868,21 +883,36 @@ def get_center_offset(omm_system):
                 online_analysis_target_error=sampler_settings.online_analysis_target_error.m,
                 online_analysis_minimum_iterations=sampler_settings.online_analysis_minimum_iterations
             )
-            # TODO - update to more verbosely pass in n_replicas and n_states, which will avoid
-            # duplication in the SAMS code path
-            sampler.setup(
-                reporter=reporter,
-                platform=platform,
-                lambda_protocol=lambdas,
-                temperature=to_openmm(self._settings.integrator_settings.temperature),
-                endstates=alchem_settings.unsampled_endstates,
-            )
         elif sampler_settings.sampler_method == "sams":
```

Comment:
I'd be tempted to throw a `.lower()` into any string comparison for luck

Reply:
Done